var/spack/repos/builtin/packages/tasmanian/package.py

```diff
@@ -31,12 +31,13 @@ class Tasmanian(CMakePackage):
     interpolation as well as parameter calibration."""
 
     homepage = 'http://tasmanian.ornl.gov'
-    url      = 'https://github.com/ORNL/TASMANIAN/archive/v5.1.tar.gz'
+    url      = 'https://github.com/ORNL/TASMANIAN/archive/v6.0.tar.gz'
     git      = 'https://github.com/ORNL/TASMANIAN.git'
 
-    version('xsdk-0.3.0', branch='master')
+    version('xsdk-0.4.0', tag='v6.0')
```

Comment:
Note: this is not needed for xsdk. I can just use `tasmanian@6.0` from xsdk/package.py

Reply:
@balay: I removed the `xsdk-0.4.0` tag, please use `tasmanian@6.0`. Thanks!